rest-server/templates/src/main/scala/com/actionml/templates/cb/CBEngine.scala

```diff
@@ -30,7 +30,7 @@ class CBEngine() extends Engine() with JsonParser {
   var algo: CBAlgorithm = _
   var params: CBEngineParams = _
 
-  override def init(json: String): Validated[ValidateError, Boolean] = {
+  override def init(json: String): Validated[ValidateError, Unit] = {
```

Comment:
This is not a big deal but for the core API I would not make such global changes without consulting first. They need to be stabilized as quickly as possible. I change them because of an algorithm or machine learning workflow need or some CLI need so ask about these first. Anything in core.templates.